assets/queries/k8s/default_namespace/query.rego

```diff
@@ -0,0 +1,30 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    
+    not document.metadata.namespace
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "issueType":		  "MissingAttribute",
+                "searchKey": 	      "metadata",
```

Comment:
Add metadata information in order to have a more specific searchKey     
 "searchKey": 	      sprintf("metadata.name=%s.metadata, [metadata.name])